src/main/java/com/igot/cb/discussion/service/impl/DiscussionServiceImpl.java

```diff
@@ -194,14 +194,17 @@ public ApiResponse createDiscussion(JsonNode discussionDetails, String token) {
             try {
                 String createdBy = discussionDetailsNode.get(Constants.CREATED_BY).asText();
                 if (CollectionUtils.isNotEmpty(userIdList)) {
-                    Map<String, Object> notificationData = Map.of(
-                            Constants.COMMUNITY_ID, discussionDetails.get(Constants.COMMUNITY_ID).asText(),
-                            Constants.DISCUSSION_ID, discussionDetails.get(Constants.DISCUSSION_ID).asText()
-                    );
-                    String firstName = helperMethodService.fetchUserFirstName(userId);
-//
-                        notificationTriggerService.triggerNotification(TAGGED_POST, ENGAGEMENT, userIdList, TITLE, firstName, notificationData);
-//
+                    List<String> filteredUserIdList = userIdList.stream()
```

Comment:
should be

userIdList.stream().filter(uniqueId -> !uniqueId.equals(createdBy)).toList();